forte/processors/stave/stave_processor.py

```diff
@@ -0,0 +1,302 @@
+        server.listen(self.configs.port)
+        IOLoop.current().start()
+
+    def _create_project(self, input_pack: DataPack):
+        with requests.Session() as session:
+
+            # Log in as admin user
+            response = session.post(f"{self._url}/api/login",
+                json={
+                    "name": self.configs.user_name,
+                    "password": self.configs.user_password
+                })
+            logger.info("%d %s", response.status_code, response.text)
+            if response.status_code != 200:
+                return
+
+            # Configure and create project
+            if self._project_id < 0:
+                response = session.post(f"{self._url}/api/projects/new",
+                    json={
+                        "type": self.configs.projectType,
+                        "name": self.configs.projectName,
+                        "ontology": self.configs.ontology.__str__(),
+                        "multiOntology": self.configs.multiOntology.__str__(),
+                        "config": self.configs.projectConfigs.__str__()
+                    })
+                logger.info("%d %s", response.status_code, response.text)
+                if response.status_code != 200:
+                    return
+                self._project_id = response.json()["id"]
+
+            # Configure and create document
+            response = session.post(f"{self._url}/api/documents/new",
+                json={
+                    "name": input_pack.pack_name,
+                    "textPack": input_pack.serialize(),
+                    "project_id": self._project_id,
+                })
+            logger.info("%d %s", response.status_code, response.text)
+            if response.status_code != 200:
+                return
+
+    @classmethod
+    def _default_project_configs(cls, ontology: Dict[str, Any]):
+        """
+        Create default project configuration based on ontology.
+        This is translated from JavaScript function `createDefaultConfig` in
+        asyml/stave/src/app/pages/Projects.tsx
+
+        Args:
+            ontology: A dictionary representing ontology.
+
+        Returns:
+            configs: A dictionary with the default config for project.
+
+        """
+        configs: Dict[str, Any] = {
+            "legendConfigs": {},
+            "scopeConfigs": {},
+            "layoutConfigs": {
+                "center-middle": "default-nlp",
+                "left": "default-meta",
+                "right": "default-attribute",
+                "center-bottom": "disable"
+            }
+        }
+
+        legend_configs: Dict[str, Any] = {}
+        entry_graph = collections.defaultdict(set)
+        annotation_entry = "forte.data.ontology.top.Annotation"
+        for entry in ontology["definitions"]:
+            entry_name = entry["entry_name"]
+
+            if entry_name == annotation_entry:
+                configs['scopeConfigs'][entry_name] = False
```

Comment:
What does this line do? I think `scopeConfig` apply to all subclass of `forte.data.ontology.top.Annotation`, here we only check for equal?